README.md

```diff
@@ -167,5 +198,29 @@ Write code to do the following:
 
 Write your code here:
 ```ruby
-# code here
+1.
+
+lamp = Lamp.create(wishes_remaining: 3)
+genie = Genie.create(name: "Genie")
+
+2.
+
+class Genie < ActiveRecord::Base
+  belongs_to :lamp
+end
```

Comment:
The question assumes the association already exists between the `Genie` and `Lamp` models. What you need to do here is make it so that a specific instance of Lamp (`lamp`) belongs to a specific instance of Genie (`genie`).